components/pages/privacy/resourceAccess/resourceAccessDrawer/index.jsx

```diff
@@ -154,31 +94,6 @@ export default function ResourceAccessDrawer({
     };
   });
 
-  const removeAccess = handleRemoveAccess({
-    accessList,
-    accessControl,
-    agentWebId,
-    setShouldUpdate,
-    setSeverity,
-    setMessage,
-    setAlertOpen,
-    onClose,
-  });
-
-  useEffect(() => {
-    if (!accessList || !accessList.length) return;
-    setAgentWebId(accessList[0]?.agent);
-  }, [accessList]);
-
-  const handleConfirmation = () => {
-    setConfirmationSetup(true);
-    setOpen(REMOVE_ACCESS_CONFIRMATION_DIALOG);
-    setIsDangerousAction(true);
-    setTitle(`Revoke access to ${resourceName}`);
```

Comment:
Can we please add a question mark at the end of this, so it says 'Revoke access to ${resourceName}?'